python/ray/scripts/scripts.py

```diff
@@ -606,7 +609,7 @@ def submit(cluster_config_file, docker, screen, tmux, stop, start,
     target = os.path.join("~", os.path.basename(script))
     rsync(cluster_config_file, script, target, cluster_name, down=False)
 
-    cmd = " ".join(["python", target] + list(script_args))
+    cmd = " ".join(["python", target] + args.split(" "))
```

Comment:
Maybe pass in the args directly as is?
```suggestion
    cmd = " ".join(["python", target, args])
```

Reply:
done!